src/geometrytools.cpp

```diff
@@ -599,10 +599,10 @@ cv::Mat ransacP3PDistortion(std::vector<cv::Point3f> scene_points,
           float(intrinsic.at<double>(1, 2));
     }
     // Run p3p
-    cv::Mat distortion_vector = (cv::Mat_<double>(1, 5) << 0, 0, 0, 0, 0);
+    distortion_vector = (cv::Mat_<double>(1, 5) << 0, 0, 0, 0, 0);
     Inliers =
         ransacP3P(scene_points, imagePointsUndis, New_Intrinsic,
-                  distortion_vector, best_R, best_T, thresh, p, it, refine);
+                  distortion_vector, best_R, best_T, thresh, it, p, refine);
```

Comment:
I inverted those because default variables should go to the end of the function declaration. I believe this change is consistent with other modifications.

Reply:
Perfect